k8s/namespaces/probate/probate-cron-smee-and-ford-extract/demo-00.yaml

```diff
@@ -11,4 +11,68 @@ spec:
       tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
       environment: demo
     job:
+      image: hmctspublic.azurecr.io/probate/back-office:prod-721a898-20220729151600 #{"$imagepolicy": "flux-system:probate-back-office"}
+      keyVaults:
+        probate:
+          secrets:
+            - name: AppInsightsInstrumentationKey
```

Comment:
Im not sure how these yamls work in cnp flux - do we really need to add them all in here?

Reply:
on K8 cron the secrets should be passed, I will try without it